module-check/src/test/clojure/clojure/core/typed/test/core_test.clj

```diff
@@ -0,0 +1,7 @@
+(ns test.core-test
+(:require [clojure.core.typed :as t]
+                [clojure.test :refer :all]
+                [test.core]))
```

Comment:
Sorry for being pedantic, but it's important for big projects to have consistent style to make it easy to understand.
1. :require should have 2 space indentation
2. second and third lines should be aligned with the first one:

```clojure
(ns test.core-test  
  (:require [clojure.core.typed :as t]
            [clojure.test :refer :all]
            [test.core]))
```